deploy/build.sh

```diff
@@ -34,6 +34,14 @@ function fix_python_recursively() {
   done
 }
 
+function md5() {
+  if which md5 >/dev/null 2>&1; then
```

Comment:
please switch the order of md5 and md5sum. The latter is GNU, so most probably will be present on linux system. Also I suggest rename the function a bit since I am not sure who will get called first if there is a system function called md5.

note there could be a false alarm when using one function in build, and use another when deploy. But chance is not high so we can live with it right now. But because of this, please log the real function we used when build and deploy so in case this happens, we can debug.